[DOC-1137][2025.2.1] Update docs for Auth Passthrough (#30133)
yugabyte/yugabyte-db: docs/content/stable/additional-features/connection-manager-ysql/ycm-setup.md

| ysql_conn_mgr_port | YSQL Connection Manager port to which clients can connect. This must be different from the PostgreSQL port set via `pgsql_proxy_bind_address`.<br>Default: 5433 |
| ysql_conn_mgr_server_lifetime | The maximum duration (in seconds) that a backend PostgreSQL connection managed by YSQL Connection Manager can remain open after creation.<br>Default: 3600 |
| ysql_conn_mgr_log_settings | Comma-separated list of log settings for YSQL Connection Manger. Can include 'log_debug', 'log_config', 'log_session', 'log_query', and 'log_stats'.<br>Default: "" |

> | ysql_conn_mgr_use_auth_backend | In "auth backend" mode, each incoming authentication request is handled by a freshly spawned postgres backend. When disabled, the "auth passthrough" mechanism is used instead, wherein backends from a pool of "control backends" are reused for all authentication requests, potentially reducing new connection acquisition latency. <br>Default: true |

Reviewer: ```suggestion
| ysql_conn_mgr_use_auth_backend | When set to true ("auth backend mode"), each incoming authentication request is handled by a freshly spawned PostgreSQL backend. Set this parameter to false to use an authentication passthrough mechanism, where backends from a pool of "control backends" are reused for all authentication requests; this can potentially reduce new connection acquisition latency. <br>Default: true |
```
